### PR TITLE
GIT_SSH wrapper should be a script

### PR DESCRIPTION
**Pull request by karasz**
GIT_SSH wrapper should be a valid script so it needs a shabang.

### CODE REVIEW

File: scripts.go [hunk 1]
```diff
@@ -116,7 +116,8 @@ trap 'rm -f /tmp/.git_ssh.$$' 0
 
 if [ "$1" = "-i" ]; then
     SSH_KEY=$2; shift; shift
-    echo "ssh -i $SSH_KEY \$@" > /tmp/.git_ssh.$$
+    echo "#!/bin/%v \n \
```
Comment (abiosoft) [inline scripts.go]: With `%v`, another parameter is introduced so you need to modify line 130/131 accordingly.

You'll also need to ensure the test passes.

**Comment by abiosoft** [comment]
What platform are you on?

**Comment by karasz** [comment]
An alpine based docker image.

**Comment by abiosoft** [comment]
For the test to pass, you need to modify git_test.go and change the string in `expectedWrapperScript` accordingly.

**Pull request merged**